frontends/web/src/components/bitboxbase/unlock.tsx

```diff
@@ -0,0 +1,131 @@
+import { Guide } from '../../components/guide/guide';
+import { AppLogo, SwissMadeOpenSource } from '../../components/icon/logo';
+import { Footer, Header } from '../../components/layout';
+import { PasswordSingleInput } from '../../components/password';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { apiPost } from '../../utils/request';
+
+interface UnlockBaseProps {
+    bitboxBaseID: string | null;
+}
+
+interface State {
+    bitboxBaseID: string | null;
+    password?: string;
+    username?: string;
+}
+
+type Props = UnlockBaseProps & TranslateProps;
+
+class UnlockBase extends Component<Props, State> {
+    constructor(props) {
+        super(props);
+        this.state = {
+            bitboxBaseID: '',
+            password: '',
+            username: 'admin',
+        };
+    }
+
+    private apiPrefix = () => {
+        return 'bitboxbases/' + this.props.bitboxBaseID;
+    }
+
+    private handleFormChange = password => {
+        this.setState({ password });
+    }
+
+    private validate = () => {
+        return this.state.password !== '';
+    }
+
+    private handleSubmit = event => {
+        event.preventDefault();
+        if (!this.validate()) {
+            return;
+        }
+        apiPost(this.apiPrefix() + '/user-authenticate', {username: 'admin', password: this.state.password })
+        .then(response => {
+            if (!response.success) {
+                // TODO: Once error codes are implemented on the base, add them with corresponding text to app.json for translation
+                alertUser(response.message);
+            }
+        });
+        this.setState({ password: '' });
+    }
+
+    public render(
+        {
+            t,
+        }: RenderableProps<Props>,
+        {
+            password,
+        }: State,
+    ) {
+        return (
+            <div class="contentWithGuide">
+                <div className="container">
+                    <Header title={<h2>{t('welcome.title')}</h2>} />
+                    <div className="innerContainer scrollableContainer">
+                        <div className="content narrow padded isVerticallyCentered">
+                            <AppLogo />
+                            <div class="box large">
+                                {
+                                    <form onSubmit={this.handleSubmit}>
+                                        <div className="m-top-default">
+                                            <PasswordSingleInput
+                                                autoFocus
+                                                id="password"
+                                                type="password"
+                                                label="Password"
+                                                placeholder="Enter your BitBoxBase password to unlock"
+                                                onValidPassword={this.handleFormChange}
```

Comment:
please put into app.json, it's hard to find those hardcoded strings afterwards.

Reply:
Which hardcoded string do you mean here?

Reply:
sorry was one line above

"Enter your BitBoxBase password to unlock"